wpilibc/src/main/native/include/frc/simulation/DCMotorSim.h

```diff
@@ -57,8 +42,8 @@ class DCMotorSim : public LinearSystemSim<2, 1, 2> {
    * @param angularPosition The new position
    * @param angularVelocity The new velocity
    */
-  void SetState(units::radian_t angularPosition,
-                units::radians_per_second_t angularVelocity);
+  void SetState(units::radian_t angularPosition = 0_rad,
```

Comment:
Why are these defaulted? This is inconsistent with Java's API.

Reply:
> Why are these defaulted? This is inconsistent with Java's API.

I don't know and I don't remember doing this.  I must have.  

Sorry about that.  Fix sent.